lib/matplotlib/axes/_base.py

```diff
@@ -263,6 +263,14 @@ def _plot_args(self, tup, kwargs):
             raise ValueError('third arg must be a format string')
         else:
             linestyle, marker, color = None, None, None
+
+        # check that n values of None are passed in
```

Comment:
"n" -> "no".  Comment could be clarified a bit:
```
Don't allow any None values; a None would be converted
to a one-element array of None, which would fail downstream.
```
The comment is probably overkill; there was never any 
reason to think None *should* be allowed, it just was
not explicitly blocked because we didn't attempt exhaustive
argument validation.